extract_wheels/lib/requirements.py

```diff
@@ -0,0 +1,45 @@
+import re
```

Comment:
μNit: alternatively would call this module `dependency_specification.py` to closely match the concerns of the pep.

Reply:
I was planning to expand the parsing out to capture more info so we can play with things like visibility. I think dependency_specification would be the superclass if we started supporting more formats. This file is intended to grow specifically for `requirements.txt`